Xamarin.Forms.Platform.Android/CollectionView/ItemContentView.cs

```diff
@@ -66,6 +66,16 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 			var width = Context.FromPixels(pixelWidth);
 			var height = Context.FromPixels(pixelHeight);
 
+			if (height == 0 && MeasureSpec.GetMode(heightMeasureSpec) == MeasureSpecMode.Unspecified)
+			{
+				height = double.PositiveInfinity;
+			}
+
+			if (width == 0 && MeasureSpec.GetMode(widthMeasureSpec) == MeasureSpecMode.Unspecified)
+			{
+				width = double.PositiveInfinity;
+			}
```

Comment:
I think here you can optimize the code a bit so as not to use `Context.FromPixels` conversions once again.
```C#
var height = MeasureSpec.GetMode(heightMeasureSpec) == MeasureSpecMode.Unspecified 
	? double.PositiveInfinity
	: Context.FromPixels(pixelHeight);

var width = MeasureSpec.GetMode(widthMeasureSpec) == MeasureSpecMode.Unspecified 
	? double.PositiveInfinity
	: Context.FromPixels(pixelWidth);
```

Reply:
👍